app/assets/javascripts/details/shell/action_bar_bridge.js

```diff
@@ -0,0 +1,116 @@
+var ActionBarBridge;
```

Comment:
@rememberlenny think of my point of view: a skilled developer who's unfamiliar with the framework you've created. If this was angular, there'd be docs online that'd tell me the purpose of the class and point to examples how to use it. But here, we don't have that. So for the sake of someone new to the code, what does the ActionBarBridge do?